Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -1,43 +1,47 @@
 ﻿<###
 # Overview:
-    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for admin roles (Owner, Contributor, UAA) in a Subscription.
+    This script is used to configure Conditional Access (CA) policy to require PIM elevation from SAW for all built-in roles except a few and all custom roles in a Subscription.
+    Following is the list of excluded built-in role definitions:
+    [ Azure Front Door Domain Contributor, Azure Front Door Domain Reader, Azure Front Door Profile Reader,
+      Azure Front Door Secret Contributor, Azure Front Door Secret Reader, Defender for Storage Data Scanner,
+      AzureML Compute Operator, Cognitive Services Usages Reader, Key Vault Crypto Service Release User ]
 
 # Control ID:
     Azure_Subscription_Configure_Conditional_Access_for_PIM
 
 # Display Name:
-    Enable policy to require PIM elevation from SAW for admin roles in Azure subscriptions.
+    Enable policy to require PIM elevation from SAW Device and SC-ALT account for critical roles (all built-in except a few mentioned above and all custom roles) in Azure subscriptions.
```

Comment:
is this display name changed - 
Enable policy to require PIM elevation from SAW for Azure roles in Azure subscriptions

Reply:
No, the control display name has not changed, I am reverting it back.